src/Model/ModelManager.php

```diff
@@ -511,15 +522,15 @@ public function collectionRemoveElement(&$collection, &$element)
     }
 
     /**
-     * method taken from PropertyPath.
+     * method taken from Symfony\Component\PropertyAccess\PropertyAccessor.
      *
      * @param string $property
      *
      * @return mixed
      */
     protected function camelize($property)
     {
-        return preg_replace(['/(^|_)+(.)/e', '/\.(.)/e'], ["strtoupper('\\2')", "'_'.strtoupper('\\1')"], $property);
+        return str_replace(' ', '', ucwords(str_replace('_', ' ', $property)));
```

Comment:
[I used the `SonataDoctrineOrmBundle` one](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/3.x/src/Model/ModelManager.php#L634). [The support of the /e modifier was removed with PHP 7.0](https://www.php.net/manual/en/function.preg-replace.php#refsect1-function.preg-replace-changelog)